pyproject.toml

```diff
@@ -0,0 +1,36 @@
+[tool.poetry]
+name = "pyband"
+version = "0.3.0"
+license="MIT"
+description= "Python library for BandChain"
+authors = ["Band Protocol"]
+readme = "README.md"
+homepage = "https://bandprotocol.com/"
+repository = "https://github.com/bandprotocol/pyband"
+documentation = "https://docs.bandchain.org/client-library/pyband/getting-started.html"
+keywords=["BAND", "BLOCKCHAIN", "ORACLE"]
+classifiers=[
+    "Development Status :: 3 - Alpha",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: MIT License",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.6",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+]
+
+[tool.poetry.dependencies]
+python = "^3.7"
+python-dateutil = "2.8.2"
+bech32 = "1.2.0"
+bip32 = "0.0.8"
+ecdsa = "0.15"
+mnemonic = "0.19"
+betterproto = "2.0.0b5"
+grpclib = "0.4.1"
+ledgerblue = "0.1.42"
+
+
```

Comment:
https://python-poetry.org/docs/managing-dependencies/#dependency-groups add testing dependency here and update test ci to install via poetry